package.json

```diff
@@ -23,7 +23,7 @@
     "@oclif/plugin-warn-if-update-available": "1.7.0",
     "bluebird": "3.5.2",
     "chalk": "4.0.0",
-    "cli-table": "github:Automattic/cli-table#master",
+    "cli-table": "github:Automattic/cli-table#7b14232ba779929e1859b267bf753c150d90a618",
```

Comment:
For what I can see from their repo, they tagged a new version 0.3.3 recently, but did not release it. 

I suppose that this is the safer way to go for the moment.

Reply:
Yeah, the release is very recent (around 2 hours ago) and it seems that it failed to build. The commit hash is the last one that worked on an `npm` project I had that already had `cli-table` installed. Also, they seemed to change the pkg name in one of the latest commits, which also seems to break stuff.

Until there's an `npm install`-able version of `cli-table` (currently `npm i cli-table` doesn't work either), I definitely think that this is the safest approach. A more holistic fix, IMHO, would be to not point to the package through github, once that's possible.